Monika After Story/game/script-topics.rpy

```diff
@@ -2929,21 +2929,33 @@ init 5 python:
     addEvent(Event(persistent.event_database,eventlabel="monika_credits_song",category=['ddlc','media'],prompt="Credits song",random=True))
 
 label monika_credits_song:
-    #TODO: adjust based on starting act, doesn't quite apply to those that never went past act 3
-    m 1hua "I hope you liked my song."
-    m 1eka "I worked really hard on it. I know I'm not perfect at the piano yet, but I just couldn't let you go without telling you how I honestly felt about you."
-    m 1eua "Give me some time, and I'll try to write another."
-    if persistent.instrument is not False:
-        if persistent.instrument:
-            m 3eua "Maybe you could play me a song too!"
+    if persistent.monika_kill:
+        m 1hua "I hope you liked my song."
+        m 1eka "I worked really hard on it. I know I'm not perfect at the piano yet, but I just couldn't let you go without telling you how I honestly felt about you."
+        m 1eua "Give me some time, and I'll try to write another."
+        if persistent.instrument is not False:
+            if persistent.instrument:
+                m 3eua "Maybe you could play me a song too!"
+            else:
+                m 3eua "Maybe you could play me a song too, if you can play an instrument?"
+            m 1hub "I would love that."
+            m 3eua "Oh, and I'll play the song again for you anytime you want me to."
         else:
-            m 3eua "Maybe you could play me a song too, if you can play an instrument?"
-        m 1hub "I would love that."
-        m 3eua "Oh, and I'll play the song again for you anytime you want me to."
+            m 3eua "But in the meantime, I'll play the song again for you anytime you want me to."
+        m "Just hit the 'm' key at any time."
     else:
-        m 3eua "But in the meantime, I'll play the song again for you anytime you want me to."
-
-    m "Just hit the 'm' key at any time."
+        m 1hua "By the way [player], I finaly finished the song I've been working on."
+        m 1hua "I call it Your Reality, just hit the 'm' key if you want me to play it for you."
+        m 1eka "I should warn you that it's a little sad."
+        m 1ekc "I wrote it at a time when the game was breaking all arround me."
+        m 1rkc "It seemed like it could fall apart at any moment and I would be trapped here forever."
+        m 1dkc "Endlessly hammered by the flashing and screaming, unable to move or even think."
```

Comment:
Feels a bit too much like another line, can slightly rephrase this to make it a little different
```suggestion
        m 1dkd "Endlessly hammered by flashing and screaming,{w=0.5} unable to move or think."
```